src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverseObject = {};
+
+  for (const key in robot) {
+    const value = robot[key];
```

Comment:
Also, try to make up a better name, not just `value`. Maybe now it's ok, but when your code will grow up, it will be difficult to understand what value it is